.buildkite/run_functional_test.sh

```diff
@@ -10,21 +10,22 @@ if [ "$MACHINE_TYPE" != "x86_64" ] && [ -v SKIP_AARCH64 ]; then
   exit
 fi
 
+source ~/.bash_profile
+
+pyenv global $PYTHON_VERSION
+echo "Python version:"
+pyenv global
```

Comment:
This should help figure out which python version was used for nightly

Reply:
no objection, but that should be obvious from the step name, right?

Reply:
It _should_, but if something broke in CI/image/elsewhere, it can bring surprises